tasks/computer-vision/image-classification/README.md

```diff
@@ -1 +1,10 @@
-# Image Classification
+# Image Classification
+
+
+Developed an Residual Attention Network where the main branch is based off ResNet18 architecture and the mask branch is a AutoEncoder (U-Net).

```

Comment:
please choose one architecture which you give better performance or you can have option to switch between architectures by having parameters. For time being I am reviewing branch which used for pull request.